app/ui.js

```diff
@@ -0,0 +1,73 @@
+// Select DOM elements to work with
+const welcomeDiv = document.getElementById("WelcomeMessage");
+const signInButton = document.getElementById("SignIn");
+const cardDiv = document.getElementById("card-div");
+const mailButton = document.getElementById("readMail");
+const profileButton = document.getElementById("seeProfile");
+const profileDiv = document.getElementById("profile-div");
+
+function showWelcomeMessage(account) {
+
+    // Reconfiguring DOM elements
+    cardDiv.classList.remove('d-none');
+    welcomeDiv.innerHTML = `Welcome ${account.name}`;
+
+    // Reconfiguring DOM elements
+    cardDiv.style.display = 'initial';
+    welcomeDiv.innerHTML = `Welcome ${account.name}`;
+    signInButton.nextElementSibling.style.display = 'none';
```

Comment:
since we have only one button, this line should go as well:
` signInButton.nextElementSibling.style.display = 'none';`